Library/Formula/lorem.rb

```diff
@@ -13,6 +13,28 @@ def install
   end
 
   test do
-    assert_equal "lorem ipsum", shell_output("#{bin}/lorem -n 2").strip
+    assert_equal "lorem ipsum", shell_output("#{bin}/lorem -n 2").strip.downcase
   end
+
+  patch :DATA
```

Comment:
Since this is an upstream commit, please add the patch as an URL to the commit as well as a comment explaining why it’s necessary. You can get a patch from any commit on GitHub by appending `.diff` to its URL. Put it above `def install`, too:

```
# This patch is necessary because…
patch do
  url "https://github.com/per9000/lorem/commit/1e3167d15b1337665a236a1e65a582ad2e3dd994.diff"
  sha256 "…"
end
```